### PR TITLE
Feature/lu 4528/ingest pck to json

### PR DESCRIPTION
**Pull request by mattprice-28**
PR, serverless lambda to convert pck files to json format and write to s3

### CODE REVIEW

File: generate_pck.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "0019:" + str(reference) + "F:0901\n" + \
+    "0001 00000000002\n" + \
+    "0011 00000011009\n" + \
+    "0012 00000311009\n" + \
+    "0678 00000000979\n" + \
+    "0679 00000000003\n" + \
+    "0146 00000000002\n"
+    n = n + 1
+    reference = reference + 1
+print(pck_file)
```
Comment (sufyaniqbal) [inline generate_pck.py]: maybe not print out the output if the file is being created?

File: lambda_function.py [hunk 2]
```diff
@@ -1,2 +1,31 @@
+import boto3
+import json
+from pck_to_json import main
+
+
```
Comment (sufyaniqbal) [inline lambda_function.py]: reference variable name continuity

File: lambda_function.py [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
 def lambda_handler(event, context):
-    print("Hello world")
+    lambdaName = "takeon-ingest-pck-to-json"
+    json_payload = json.loads(event['Records'][0]['body'])
+    pck_title = (str(json_payload['Records'][0]['s3']['object']['key']))
+    survey_code = (str(pck_title[5:8]))
+    # print('Survey Code: ' + survey_code)
```
Comment (sufyaniqbal) [inline lambda_function.py]: delete?

File: pck_to_json.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+            form["formtype"] = parts[0]
+            form["reference"] = parts[1][:-1]
+            form["period"] = "20" + parts[2]
+            form["survey"] = survey
+        elif " " in batch[i]:  # Strip response data
+            qnum = batch[i][:4]
+            qval = batch[i][5:16]
+            responses[qnum] = qval
+        else:
+            logger.error("this is error")  # Log error with rabbitmq
```
Comment (RyanAldy) [inline pck_to_json.py]: Remove references to the logger and raise an error to the calling method?

**Pull request merged**